component/admin/controllers/icalrepeat.php

```diff
@@ -338,7 +338,7 @@ function apply()
 
 	function select()
 	{
-		JSession::checkToken('default') or jexit('Invalid Token');
+		JSession::checkToken('get') or jexit('Invalid Token');
```

Comment:
icalevent.select is called as a GET in some places and a POST in others so we should use

JSession::getToken('request');